operator/webhooks/pod_mutating_webhook.go

```diff
@@ -128,9 +137,9 @@ func (a *PodMutatingWebhook) InjectDecoder(d *admission.Decoder) error {
 	return nil
 }
 
-func (a *PodMutatingWebhook) isKeptnAnnotated(pod *corev1.Pod) (bool, error) {
-	workload, gotWorkloadAnnotation := getLabelOrAnnotation(pod, common.WorkloadAnnotation, common.K8sRecommendedWorkloadAnnotations)
-	version, gotVersionAnnotation := getLabelOrAnnotation(pod, common.VersionAnnotation, common.K8sRecommendedVersionAnnotations)
+func (a *PodMutatingWebhook) isPodAnnotated(pod *corev1.Pod) (bool, error) {
+	workload, gotWorkloadAnnotation := getLabelOrAnnotation(&pod.ObjectMeta, common.WorkloadAnnotation, common.K8sRecommendedWorkloadAnnotations)
```

Comment:
why getLabelOrAnnotation takes a pointer now?

Reply:
basically all funcs in the webhook work with a pod pointer. i just made it consistent :)